modules/registrars/openprovider/Controllers/System/DomainSuggestionsController.php

```diff
@@ -67,14 +74,56 @@ public function get($params)
         if (isset($suggestionSettings['sensitive']) && $suggestionSettings['sensitive'] == 'on')
             $args['sensitive'] = 1;
 
-        if (isset($suggestionSettings['suggestTlds']) && count($suggestionSettings['suggestTlds']) > 0)
+        if (isset($suggestionSettings['suggestTlds']) && count($suggestionSettings['suggestTlds']) > 0) {
             $args['tlds'] = array_map(function ($tld) {
                 return mb_substr($tld, 1);
             }, explode(',', $suggestionSettings['suggestTlds']));
+        }
+
+        $placementLogin = Configuration::get('placementPlusAccount');
+        $placementPassword = Configuration::get('placementPlusPassword');
+        $isTestModeEnabled = $params['test_mode'] == 'on';
+
+        if ($placementLogin && $placementPassword) {
+            $encodedDomain = urlencode($params['searchTerm']);
+
+            $countSuggestedDomainsFromPlacementPlus = $isTestModeEnabled ?
```

Comment:
also may be move it to PlacementPlus::getSuggestionDomain()? as I see this value is static by environment

Reply:
ok it can be stay here